uni/lib/Makefile

```diff
@@ -7,6 +7,7 @@ UFILES=gui.u file_dlg.u db.u \
  address.u base64handler.u basicclasscoding.u class.u classcoding.u \
  cltable.u comparator.u complex.u compoundedit.u contentdisposition.u contenttype.u \
  cvsuser.u cvsutil.u decode.u encode.u encodinghandler.u error.u format.u \
+ FxPt.u \
```

Comment:
any reason to add a new line for this? can be added the next line, it got room.

Reply:
Agreed